tests/unit/amo/reducers/testFeatured.js

```diff
@@ -20,7 +20,9 @@ describe('featured reducer', () => {
   describe('FEATURED_GET', () => {
     it('sets the initialState', () => {
       const { addonType, loading, results } = featured(
-        initialState, actions.getFeatured({ addonType: 'theme' }));
+        initialState, actions.getFeatured({
+          addonType: 'theme', errorHandlerId: 'some-error-handler',
```

Comment:
(Though I see now this was probably my bad originally, very sorry.)

![shame-cube](https://user-images.githubusercontent.com/90871/28392030-2e38f07a-6cd8-11e7-988b-8dcecd08d577.gif)